.tools/psalm/baseline-taint.xml

```diff
@@ -9,158 +9,11 @@
-    </TaintedHtml>
-  </file>
-  <file src="redaxo/src/addons/mediapool/pages/structure.php">
-    <TaintedHtml occurrences="2">
-      <code>rex_view::error($error)</code>
-      <code>rex_view::info($success)</code>
-    </TaintedHtml>
-  </file>
-  <file src="redaxo/src/addons/mediapool/pages/sync.php">
-    <TaintedHtml occurrences="2">
-      <code>rex_view::error(implode('&lt;br /&gt;', $error))</code>
-      <code>rex_view::info(implode('&lt;br /&gt;', $success))</code>
-    </TaintedHtml>
-  </file>
-  <file src="redaxo/src/addons/mediapool/pages/upload.php">
-    <TaintedHtml occurrences="1">
-      <code>rex_view::error(rex_i18n::msg('csrf_token_invalid'))</code>
-    </TaintedHtml>
-  </file>
-  <file src="redaxo/src/addons/metainfo/pages/field.php">
-    <TaintedHtml occurrences="2">
-      <code>rex_view::error(rex_i18n::msg('minfo_field_error_deleted'))</code>
-      <code>rex_view::success(rex_i18n::msg('minfo_field_successfull_deleted'))</code>
-    </TaintedHtml>
-  </file>
-  <file src="redaxo/src/addons/phpmailer/pages/config.php">
-    <TaintedHtml occurrences="4">
-      <code>rex_view::success($addon-&gt;i18n('archive_deleted'))</code>
-      <code>rex_view::success($message)</code>
-      <code>rex_view::warning($warning)</code>
-      <code>rex_view::warning($warning)</code>
-    </TaintedHtml>
-  </file>
-  <file src="redaxo/src/addons/structure/pages/index.php">
-    <TaintedHtml occurrences="2">
-      <code>rex_view::error('Oooops. Your clang ids start with &lt;code&gt;0&lt;/code&gt;. Looks like a broken REDAXO 4.x to 5.x upgrade. Please update all your database tables, php code (if there are any hard coded clang ids) aswell as additional configurations in add-ons, e.g. YRewrite. You may start with updating those tables: &lt;code&gt;rex_article&lt;/code&gt;, &lt;code&gt;rex_article_slice&lt;/code&gt;, &lt;code&gt;rex_clang&lt;/code&gt;, by increasing every clang id &lt;code&gt;+ 1&lt;/code&gt;.')</code>
-      <code>rex_view::error('You have no permission to access this area')</code>
-    </TaintedHtml>
-  </file>
   <file src="redaxo/src/addons/structure/plugins/content/lib/article_content_base.php">
     <TaintedHtml occurrences="1">
       <code>$articleContent</code>
     </TaintedHtml>
   </file>
-  <file src="redaxo/src/addons/structure/plugins/content/pages/content.php">
-    <TaintedHtml occurrences="2">
-      <code>rex_view::error(rex_i18n::msg('article_doesnt_exist'))</code>
-      <code>rex_view::warning(rex_i18n::msg('no_rights_to_edit'))</code>
-    </TaintedHtml>
-  </file>
-  <file src="redaxo/src/addons/structure/plugins/content/pages/templates.php">
-    <TaintedHtml occurrences="1">
-      <code>rex_view::error(rex_i18n::msg('csrf_token_invalid'))</code>
-    </TaintedHtml>
-  </file>
   <file src="redaxo/src/addons/structure/plugins/history/fragments/history/layer.php">
     <TaintedHtml occurrences="4">
       <code>$this-&gt;getVar('content1iframe')</code>
```

Comment:
Müsste man lt. verblieber baseline auch getVar() so markieren? (Geraten)

Reply:
glaube nicht. bei fragment->getvar ist eher das problem, dass es unmöglich ist, zum zugehörigen setvar-aufruf zu verknüpfen, da an völlig anderer stelle (getVar in fragment, setVar an fragment-aufruf-stelle). da ist mir bisher noch keine lösung für eingefallen.